system/security/zcrypt/base64.cpp

```diff
@@ -98,7 +98,7 @@ ZBuffer& base64_decode(int inlen, const char *in, ZBuffer& data)
     {
         for(int i = 0; i < 64; ++i)
         {
-            BASE64_dec[BASE64_enc[i]] = i;
+            BASE64_dec[(int) BASE64_enc[i]] = i;
```

Comment:
Should probably cast to unsigned char instead (won\t cause a problem in practice though)